### PR TITLE
chore: add license

### PR DESCRIPTION
**Pull request by bonustrack**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "sx-monorepo",
+  "license": "MIT",
```
Comment (Sekhmet) [inline package.json:3]: This shouldn't really matter as we don't publish root package (it's private).

**Review by Sekhmet — APPROVED** [review]
utACK

File: package.json (hunk 1 above)
Comment (bonustrack) [reply package.json:3]: It's just so it show up on GitHub

Comment (Sekhmet) [reply package.json:3]: It reads it from LICENSE/LICENDE.md file I belive.

Comment (bonustrack) [reply package.json:3]: Right, then maybe it can help if someone open this file

**Pull request merged**